api/sendMail.php

```diff
@@ -1,29 +1,56 @@
 <?php
+
 header('Access-Control-Allow-Origin: *');
```

Comment:
you should not allow all, this is not secure.

Reply:
replace * with domain name